agent/exploits/cve_2021_33044.py

```diff
@@ -0,0 +1,95 @@
+"""Agent Asteroid implementation for CVE-2021-33044"""
+
+import re
+import json
+import requests
+import logging
+import urllib3
+
+from agent.exploits import webexploit
+from agent import definitions
+from agent import exploits_registry
+
+VULNERABILITY_TITLE = "Dahua IPC/VTH/VTO devices auth bypass"
+VULNERABILITY_REFERENCE = "CVE-2021-33044"
+VULNERABILITY_DESCRIPTION = (
+    "The identity authentication bypass vulnerability found in some Dahua products during the login process. "
+    "Attackers can bypass device identity authentication by constructing malicious data packets."
+)
+
+RISK_RATING = "CRITICAL"
+
+logger = logging.getLogger(__name__)
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def _send_payload(target: definitions.Target) -> bytes | None:
```

Comment:
missing docstring

Reply:
@Zakaria-Latif 
It's a private method we don't really need.

Reply:
No docstring in private methods